apps/server-asset-sg/prisma/migrations/20241001062933_add_legal_files/migration.sql

```diff
@@ -0,0 +1,37 @@
+-- Cleanup workgroup migration.
+ALTER TABLE "asset" DROP CONSTRAINT "asset_workgroup_id_fkey";
+ALTER TABLE "asset" ADD CONSTRAINT "asset_workgroup_id_fkey" FOREIGN KEY ("workgroup_id") REFERENCES "workgroup"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
+ALTER TABLE "asset" ALTER COLUMN "workgroup_id" SET NOT NULL;
+
+-- CreateEnum
+CREATE TYPE "FileType" AS ENUM ('normal', 'legal');
+
+-- AlterTable
+ALTER TABLE "file" RENAME COLUMN "file_id" TO "id";
+ALTER TABLE "file" RENAME COLUMN "file_date" TO "last_modified_at";
```

Comment:
is there really a concept of modifiying a file? isnt there just the upload/creation date?

Reply:
This is just the file's modification date, and we never change it after the initial upload. The field is never used, so IMO we can also remove it.